sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -288,9 +298,13 @@ export class SenderImpl implements Sender {
       sequenceNumbers
     );
 
-    return this._context.managementClient!.cancelScheduledMessages(sequenceNumbers);
+    return this._context.managementClient!.cancelScheduledMessages(
+      sequenceNumbers,
+      this._senderOptions
+    );
   }
 
+  // TO DO - No need to retry close() ???
```

Comment:
Minor nit - do you mind changing these "TO DO" items to be "TODO" just to make them easier to find later?